packages/fluentui/react-northstar/src/components/TextArea/TextArea.tsx

```diff
@@ -44,9 +45,16 @@ export interface TextAreaProps extends UIComponentProps, ChildrenComponentProps
 
   /** A textarea can take the width of its container. */
   fluid?: boolean;
+
+  /** A textarea can be set to automatically adjust the height */
+  autoAdjustHeight?: boolean;
```

Comment:
I called it `autoAdjustHeight` to be similar to Fabric and it was also as the user named it when requiring it.... But I wonder if `autoResize` wouldn't be a better name

Reply:
I would vote for `autoResize` it's what I would search for if I need behavior like this.